src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+@import "./utils/variables";
+
 body {
   margin: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
You should center just the calendaro , but wrap it first in an element of the page like page__calendar because you can't apply margin to blocks , then center it with margin 0 auto